experimental/dds/tree2/src/test/shared-tree/opSize.bench.ts

```diff
@@ -529,13 +529,48 @@ describe("Op Size", () => {
 	});
 
 	describe("Insert, Delete & Edit Nodes", () => {
+		const oneThirdNodeCount = Math.floor(BENCHMARK_NODE_COUNT * (1 / 3));
+		const seventyPercentCount = BENCHMARK_NODE_COUNT * 0.7;
+		const fifteenPercentCount = BENCHMARK_NODE_COUNT * 0.15;
+
+		type OpKindDistribution = {
+			+readonly [OpKind in keyof typeof Operation]: number;
```

Comment:
```suggestion
			readonly [OpKind in keyof typeof Operation]: number;
```

Reply:
Just curious: do we have recommended guidance for keeping or omitting the `+`? I have 0 opinion, just wondering.

Reply:
I don't think we do. I also am fine with either. I just put that as a suggestion in case you thought the `+` was required.

Reply:
Nope, I usually just err on the side of whatever is more explicit.